client/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/kubernetes-csi/external-snapshotter/client/v2
+
+go 1.12
```

Comment:
does this need to be updated?

Reply:
Hi @msau42  could you please clarify if i am not wrong ,are you mentionig here to update go version to 1.13 ?

Reply:
Other sidecars are using go 1.13 so we should update to 1.13 as well, but it's fine if you update go version in a separate PR .  This PR already has lots of changes.